supervisor/api/host.py

```diff
@@ -101,11 +99,7 @@ def shutdown(self, request):
     @api_process
     def reload(self, request):
         """Reload host data."""
-        return asyncio.shield(
-            asyncio.wait(
-                [self.sys_host.reload(), self.sys_resolution.evaluate.evaluate_system()]
```

Comment:
I feel like a re-evaluation after a host reload still makes sense no? Maybe we just queue an async task to do that and don't wait on it? It doesn't affect the response so we don't want to hold up the user but it is still an event that could affect issues and suggestions.

Reply:
API should only do what he is proposing to do. The Frontend can decide how they will call the functions for the user presentation